### ISSUE TITLE
AdvaImg: Allocation conflict at "plugingif.cpp, ~StringTable - line 191 and line 194" (by BoundsChecker)

### ISSUE THREAD

**Issue by georgehazan** [miranda-ng/miranda-ng#492]
**Reported by Corak on 30 Dec 2015 22:48 UTC**
line 191:
Allocation Conflict: Attempting to call global_operator_delete_array on 0x04BD18D8 (Miranda32.exe); pointer was allocated by global_operator_new.

![3dd55827ebd3](https://cloud.githubusercontent.com/assets/5046855/21158379/11f81bca-c18e-11e6-9c1d-6941e446f9b4.png)

![cd5496acadcc](https://cloud.githubusercontent.com/assets/5046855/21158381/154981ce-c18e-11e6-9a79-1cd914d028eb.png)

line 194:
Allocation Conflict: Attempting to call global_operator_delete_array on 0x10940040 (MSVCP100D.DLL); pointer was allocated by global_operator_new.

![ec71013b1169](https://cloud.githubusercontent.com/assets/5046855/21158404/2c5248ec-c18e-11e6-8f28-d4944cd3d7be.png)

![07c267e09ab4](https://cloud.githubusercontent.com/assets/5046855/21158408/2f5efaee-c18e-11e6-84d3-673f6764d7fb.png)

```
ErrorCategory="Allocation Conflict"
Type="Allocation conflict"
FullLocation="AdvaImg.dll, plugingif.cpp, ~StringTable - line 191"
Module="AdvaImg.dll"
ModuleFullPath="d:\Soft\Network\IM\Miranda\MirandaNG\Plugins\AdvaImg.dll"
FileName="d:\soft\network\im\miranda\src\trunk\plugins\advaimg\src\freeimage\plugingif.cpp"
FunctionName="~StringTable"
LineNumber="191"
Message="Allocation Conflict: Attempting to call global_operator_delete_array on 0x04C3DB70; pointer was allocated by global_operator_new., Interesting Location: plugingif.cpp, line 191 (AdvaImg.dll)"
/>
```

```
ErrorCategory="Allocation Conflict"
Type="Allocation conflict"
FullLocation="AdvaImg.dll, plugingif.cpp, ~StringTable - line 194"
Module="AdvaImg.dll"
ModuleFullPath="d:\Soft\Network\IM\Miranda\MirandaNG\Plugins\AdvaImg.dll"
FileName="d:\soft\network\im\miranda\src\trunk\plugins\advaimg\src\freeimage\plugingif.cpp"
FunctionName="~StringTable"
LineNumber="194"
Message="Allocation Conflict: Attempting to call global_operator_delete_array on 0x10540040; pointer was allocated by global_operator_new., Interesting Location: plugingif.cpp, line 194 (AdvaImg.dll)"
```

**Comment by georgehazan** [comment]
**Modified by Corak on 30 Dec 2015 23:00 UTC**

**Comment by georgehazan** [comment]
**Modified by Corak on 30 Dec 2015 23:02 UTC**

**Comment by georgehazan** [comment]
**Modified by Corak on 31 Dec 2015 18:22 UTC**

**Comment by georgehazan** [comment]
fixed 30.05.2012 21:27:49
Commit hash:	88708cffa15662dcd2755fce699112d24a10a087
